stack.c

```
@@ -46,6 +49,32 @@ int mill_max_cached_stacks = 64;
 static int mill_num_cached_stacks = 0;
 static struct mill_slist mill_cached_stacks = {0};
 
+static void *mill_allocstackmem(size_t stack_size) {
+#if MILL_ENABLE_STACK_GUARD
+    long pagesize = sysconf(_SC_PAGE_SIZE);
+    if (!(pagesize > 0)) {
+        return NULL;
+    }
+    mill_assert(mill_stack_size > pagesize);
```

Or just let it fail in this assert...